src/fidesops/task/graph_task.py

```diff
@@ -123,6 +123,11 @@ def generate_dry_run_query(self) -> str:
         """Type-specific query generated for this traversal_node."""
         return self.connector.dry_run_query(self.traversal_node)
 
+    def can_write_data(self) -> bool:
+        """Checks if the relevant ConnectionConfig has been granted "write" access to its data"""
+        connection_config = self.connector.configuration
```

Comment:
does `self.connector.configuration` have a schema or is it arbitrary keys?

Reply:
good catch, thanks